app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is correctly defined as a class attribute, but ensure that it is used consistently throughout the class to store instances by their name as required by the task description.